easybuild/easyconfigs/s/scanpy/scanpy-1.8.1-foss-2020b.eb

```diff
@@ -0,0 +1,69 @@
+easyblock = 'PythonBundle'
+
+name = 'scanpy'
+version = '1.8.1'
+
+homepage = 'https://scanpy.readthedocs.io/en/stable/'
+description = """Scanpy is a scalable toolkit for analyzing single-cell gene expression data built
+ jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference
+ and differential expression testing. The Python-based implementation efficiently deals with
+ datasets of more than one million cells.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'openmp': True}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('h5py', '3.1.0'),
+    ('networkx', '2.5'),
+    ('numba', '0.53.1'),
+    ('PyTables', '3.6.1'),
+    ('statsmodels', '0.12.1'),
+    ('scikit-learn', '0.23.2'),
+    ('Seaborn', '0.11.1'),
+    ('tqdm', '4.56.2'),
+    ('leidenalg', '0.8.3'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('natsort', '7.1.1', {
+        'checksums': ['00c603a42365830c4722a2eb7663a25919551217ec09a243d3399fa8dd4ac403'],
+    }),
```

Comment:
Does the comment from the `2021a` version about `xlrd` not apply here too?

Reply:
`Python/3.8.6` uses `xlrd v1.2.0`:
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-3.8.6-GCCcore-10.2.0.eb#L190